test/editions/EditionTemplates.scala

```diff
@@ -15,58 +13,61 @@ class editionTemplateTest extends FreeSpec with Matchers {
   // Currently not testing prefills!
   object TestCapi extends Capi {
     override def getPreviewHeaders(headers: Map[String, String], url: String): Seq[(String, String)] = Seq.empty[(String, String)]
-    override def getPrefillArticles(issueDate: LocalDate, capiPrefillQuery: CapiPrefillQuery, currentPageCodes: List[String]): Future[SearchResponse] = Future.successful(null)
-    override def getPrefillArticleItems(issueDate: LocalDate, capiPrefillQuery: CapiPrefillQuery): Future[List[Prefill]] = Future.successful(Nil)
+
+    override def getPrefillArticles(prefillParams: PrefillParamsAdapter, currentPageCodes: List[String]): Future[SearchResponse] = Future.successful(null)
+
+    override def getPrefillArticleItems(prefillParams: PrefillParamsAdapter): Future[List[Prefill]] = Future.successful(Nil)
   }
+
   val templating = new EditionsTemplating(TestEdition.templates, TestCapi)
 
-//  "createEdition" - {
```

Comment:
Not for this PR, but this is of no value commented out. We should either fix the test, or delete this.

Reply:
agree, i was planning to do that later anyway